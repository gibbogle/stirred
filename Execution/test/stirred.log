msg: OS is Windows
msg: Executing with OpenMP
msg:  inputfile:  tarlox.inp
msg:  outputfile: pest.out
msg: ReadCellParams new
 spcrad_value:    200.000000000000     
 Ndrugs_used:            1
msg:  ReadDrugData
im, membrane_diff_in, out:    0   0.123E-09   0.500E-12
Kmet0,Kd read:    1   0   1  0.0000  0.0000
Kmet0,Kd read:    1   0   2  2.0000  0.0000
im, membrane_diff_in, out:    1   0.300E-11   0.300E-11
Kmet0,Kd read:    1   1   1  0.0000  0.0920
Kmet0,Kd read:    1   1   2  0.0000  0.0920
 drug:            1 DNB  Tarlox          
 nmetabolites:            1
 Ndrugs_used:            1
msg:  ReadProtocol:
DRUG                                                            
Tarlox                                                          
 ndrugs_used:            1
 Tarlox          Tarlox          
 ichemo:            5
define DRUG_EVENT: volume, O2conc:   1   0.050 580.000
define MEDIUM_EVENT: volume:   2  10.000 580.000
msg:  nevents:            2
 Type 1 cells
msg: shape, sigma:   0.1200E+01  0.1200E+01  0.1823E+00  0.1823E+00
msg: Median, mean divide time:   0.2400E+02  0.2400E+02  0.2440E+02  0.2440E+02
 
GUI version: GUI2.0      
DLL version: DLL2.0      
 
msg: nsteps, NT_CONC, DELTA_T:    259     1  100.
msg: NT_DISPLAY, DT_DISPLAY:      1  100.
msg: did ReadCellParams
msg:  ncpu:      1637616
msg: OPENMP defined
msg: Requested Mnodes: **
msg: Machine processors:  8
msg: Max threads available:  8
msg: Setting Mnodes = max thread count:  8
msg:  Threads, max:            8           8
msg:  Threads, max:            8           8
msg:  Threads, max:            8           8
msg:  Threads, max:            8           8
msg:  Threads, max:            8           8
msg:  Threads, max:            8           8
msg:  Threads, max:            8           8
msg:  Threads, max:            8           8
msg: did omp_initialisation
msg:  npar =            8        1234        5678
msg: did deallocation
msg:  Initial count, max_nlist:     54000000       10000
msg: did ArrayInitialisation
msg: Startup procedures have been executed: initial T cell count:      0
Event:      0  1      0.      0.  5    0.0500 1100.0000
msg: DRUG_EVENT: volume, conc:       0.0500   1100.0000     55.0000
 
 MediumChange:
Ve:   0.0500
Ce: 580.0000  5.5000  0.0000  0.0000********  0.0000  0.0000  0.0000  0.0000  0.0000  0.0000  0.0000
total_volume, Vcells, Vm, Vr:  10.0000  0.1229  9.8771  0.0000
mass:  29.0000  0.2750  0.0000  0.0000 55.0000  0.0000  0.0000  0.0000  0.0000  0.0000  0.0000  0.0000
New total_volume (total + Ve):  10.0500
Volume of medium (total - Vcells):   9.9271
Conc:   2.9213  0.0277  0.0000  0.0000  5.5404  0.0000  0.0000  0.0000  0.0000  0.0000  0.0000  0.0000
idrug,iparent,im,ichemo,Cmedium:   1  5  0  5   0.554E+01
idrug,iparent,im,ichemo,Cmedium:   1  5  1  6   0.000E+00
 
DrugSolver: IC,EC:    0.000E+00   0.554E+01
DrugSolver: IC,EC:    0.000E+00   0.554E+01
DrugSolver: IC,EC:    0.490E+01   0.548E+01
DrugSolver: IC,EC:    0.490E+01   0.548E+01
DrugSolver: IC,EC:    0.972E+01   0.542E+01
DrugSolver: IC,EC:    0.972E+01   0.542E+01
DrugSolver: IC,EC:    0.145E+02   0.536E+01
DrugSolver: IC,EC:    0.145E+02   0.536E+01
DrugSolver: IC,EC:    0.192E+02   0.530E+01
DrugSolver: IC,EC:    0.192E+02   0.530E+01
DrugSolver: IC,EC:    0.238E+02   0.525E+01
DrugSolver: IC,EC:    0.238E+02   0.525E+01
msg: istep,hour,IC,EC:      0   0.0   0.283E+02   0.519E+01   0.367E+00   0.295E-03
DrugSolver: IC,EC:    0.283E+02   0.519E+01
DrugSolver: IC,EC:    0.283E+02   0.519E+01
DrugSolver: IC,EC:    0.542E+02   0.487E+01
DrugSolver: IC,EC:    0.542E+02   0.487E+01
DrugSolver: IC,EC:    0.779E+02   0.457E+01
DrugSolver: IC,EC:    0.779E+02   0.457E+01
DrugSolver: IC,EC:    0.996E+02   0.430E+01
DrugSolver: IC,EC:    0.996E+02   0.430E+01
DrugSolver: IC,EC:    0.119E+03   0.405E+01
DrugSolver: IC,EC:    0.119E+03   0.405E+01
DrugSolver: IC,EC:    0.138E+03   0.383E+01
DrugSolver: IC,EC:    0.138E+03   0.383E+01
DrugSolver: IC,EC:    0.154E+03   0.362E+01
DrugSolver: IC,EC:    0.154E+03   0.362E+01
DrugSolver: IC,EC:    0.169E+03   0.343E+01
DrugSolver: IC,EC:    0.169E+03   0.343E+01
DrugSolver: IC,EC:    0.183E+03   0.326E+01
DrugSolver: IC,EC:    0.183E+03   0.326E+01
DrugSolver: IC,EC:    0.196E+03   0.310E+01
DrugSolver: IC,EC:    0.196E+03   0.310E+01
DrugSolver: IC,EC:    0.207E+03   0.295E+01
DrugSolver: IC,EC:    0.207E+03   0.295E+01
DrugSolver: IC,EC:    0.218E+03   0.282E+01
DrugSolver: IC,EC:    0.218E+03   0.282E+01
DrugSolver: IC,EC:    0.227E+03   0.269E+01
DrugSolver: IC,EC:    0.227E+03   0.269E+01
DrugSolver: IC,EC:    0.236E+03   0.258E+01
DrugSolver: IC,EC:    0.236E+03   0.258E+01
DrugSolver: IC,EC:    0.244E+03   0.248E+01
DrugSolver: IC,EC:    0.244E+03   0.248E+01
DrugSolver: IC,EC:    0.251E+03   0.238E+01
DrugSolver: IC,EC:    0.251E+03   0.238E+01
DrugSolver: IC,EC:    0.258E+03   0.230E+01
DrugSolver: IC,EC:    0.258E+03   0.230E+01
DrugSolver: IC,EC:    0.264E+03   0.222E+01
DrugSolver: IC,EC:    0.264E+03   0.222E+01
msg: istep,hour,IC,EC:     18   0.5   0.270E+03   0.215E+01   0.160E+01   0.167E-01
DrugSolver: IC,EC:    0.270E+03   0.215E+01
DrugSolver: IC,EC:    0.270E+03   0.215E+01
DrugSolver: IC,EC:    0.275E+03   0.208E+01
DrugSolver: IC,EC:    0.275E+03   0.208E+01
DrugSolver: IC,EC:    0.279E+03   0.202E+01
DrugSolver: IC,EC:    0.279E+03   0.202E+01
DrugSolver: IC,EC:    0.284E+03   0.196E+01
DrugSolver: IC,EC:    0.284E+03   0.196E+01
DrugSolver: IC,EC:    0.287E+03   0.191E+01
DrugSolver: IC,EC:    0.287E+03   0.191E+01
DrugSolver: IC,EC:    0.291E+03   0.186E+01
DrugSolver: IC,EC:    0.291E+03   0.186E+01
DrugSolver: IC,EC:    0.294E+03   0.182E+01
DrugSolver: IC,EC:    0.294E+03   0.182E+01
DrugSolver: IC,EC:    0.297E+03   0.178E+01
DrugSolver: IC,EC:    0.297E+03   0.178E+01
DrugSolver: IC,EC:    0.299E+03   0.174E+01
DrugSolver: IC,EC:    0.299E+03   0.174E+01
DrugSolver: IC,EC:    0.302E+03   0.171E+01
DrugSolver: IC,EC:    0.302E+03   0.171E+01
DrugSolver: IC,EC:    0.304E+03   0.168E+01
DrugSolver: IC,EC:    0.304E+03   0.168E+01
DrugSolver: IC,EC:    0.306E+03   0.165E+01
DrugSolver: IC,EC:    0.306E+03   0.165E+01
DrugSolver: IC,EC:    0.307E+03   0.162E+01
DrugSolver: IC,EC:    0.307E+03   0.162E+01
DrugSolver: IC,EC:    0.309E+03   0.160E+01
DrugSolver: IC,EC:    0.309E+03   0.160E+01
DrugSolver: IC,EC:    0.310E+03   0.158E+01
DrugSolver: IC,EC:    0.310E+03   0.158E+01
DrugSolver: IC,EC:    0.312E+03   0.156E+01
DrugSolver: IC,EC:    0.312E+03   0.156E+01
DrugSolver: IC,EC:    0.313E+03   0.154E+01
DrugSolver: IC,EC:    0.313E+03   0.154E+01
DrugSolver: IC,EC:    0.314E+03   0.152E+01
DrugSolver: IC,EC:    0.314E+03   0.152E+01
msg: istep,hour,IC,EC:     36   1.0   0.315E+03   0.151E+01   0.161E+01   0.196E-01
DrugSolver: IC,EC:    0.315E+03   0.151E+01
DrugSolver: IC,EC:    0.315E+03   0.151E+01
DrugSolver: IC,EC:    0.315E+03   0.149E+01
DrugSolver: IC,EC:    0.315E+03   0.149E+01
DrugSolver: IC,EC:    0.316E+03   0.148E+01
DrugSolver: IC,EC:    0.316E+03   0.148E+01
DrugSolver: IC,EC:    0.317E+03   0.146E+01
DrugSolver: IC,EC:    0.317E+03   0.146E+01
DrugSolver: IC,EC:    0.317E+03   0.145E+01
DrugSolver: IC,EC:    0.317E+03   0.145E+01
DrugSolver: IC,EC:    0.318E+03   0.144E+01
DrugSolver: IC,EC:    0.318E+03   0.144E+01
DrugSolver: IC,EC:    0.318E+03   0.143E+01
DrugSolver: IC,EC:    0.318E+03   0.143E+01
DrugSolver: IC,EC:    0.318E+03   0.142E+01
DrugSolver: IC,EC:    0.318E+03   0.142E+01
DrugSolver: IC,EC:    0.319E+03   0.141E+01
DrugSolver: IC,EC:    0.319E+03   0.141E+01
DrugSolver: IC,EC:    0.319E+03   0.141E+01
DrugSolver: IC,EC:    0.319E+03   0.141E+01
DrugSolver: IC,EC:    0.319E+03   0.140E+01
DrugSolver: IC,EC:    0.319E+03   0.140E+01
DrugSolver: IC,EC:    0.319E+03   0.139E+01
DrugSolver: IC,EC:    0.319E+03   0.139E+01
DrugSolver: IC,EC:    0.319E+03   0.139E+01
DrugSolver: IC,EC:    0.319E+03   0.139E+01
DrugSolver: IC,EC:    0.319E+03   0.138E+01
DrugSolver: IC,EC:    0.319E+03   0.138E+01
DrugSolver: IC,EC:    0.319E+03   0.138E+01
DrugSolver: IC,EC:    0.319E+03   0.138E+01
DrugSolver: IC,EC:    0.319E+03   0.137E+01
DrugSolver: IC,EC:    0.319E+03   0.137E+01
DrugSolver: IC,EC:    0.319E+03   0.137E+01
DrugSolver: IC,EC:    0.319E+03   0.137E+01
DrugSolver: IC,EC:    0.319E+03   0.136E+01
DrugSolver: IC,EC:    0.319E+03   0.136E+01
msg: istep,hour,IC,EC:     54   1.5   0.319E+03   0.136E+01   0.161E+01   0.200E-01
DrugSolver: IC,EC:    0.319E+03   0.136E+01
DrugSolver: IC,EC:    0.319E+03   0.136E+01
DrugSolver: IC,EC:    0.319E+03   0.135E+01
DrugSolver: IC,EC:    0.319E+03   0.135E+01
DrugSolver: IC,EC:    0.319E+03   0.135E+01
DrugSolver: IC,EC:    0.319E+03   0.135E+01
DrugSolver: IC,EC:    0.319E+03   0.134E+01
DrugSolver: IC,EC:    0.319E+03   0.134E+01
DrugSolver: IC,EC:    0.319E+03   0.134E+01
DrugSolver: IC,EC:    0.319E+03   0.134E+01
DrugSolver: IC,EC:    0.319E+03   0.134E+01
DrugSolver: IC,EC:    0.319E+03   0.134E+01
DrugSolver: IC,EC:    0.318E+03   0.134E+01
DrugSolver: IC,EC:    0.318E+03   0.134E+01
DrugSolver: IC,EC:    0.318E+03   0.133E+01
DrugSolver: IC,EC:    0.318E+03   0.133E+01
DrugSolver: IC,EC:    0.318E+03   0.133E+01
DrugSolver: IC,EC:    0.318E+03   0.133E+01
DrugSolver: IC,EC:    0.318E+03   0.133E+01
DrugSolver: IC,EC:    0.318E+03   0.133E+01
DrugSolver: IC,EC:    0.318E+03   0.132E+01
DrugSolver: IC,EC:    0.318E+03   0.132E+01
DrugSolver: IC,EC:    0.317E+03   0.132E+01
DrugSolver: IC,EC:    0.317E+03   0.132E+01
DrugSolver: IC,EC:    0.317E+03   0.132E+01
DrugSolver: IC,EC:    0.317E+03   0.132E+01
DrugSolver: IC,EC:    0.317E+03   0.132E+01
DrugSolver: IC,EC:    0.317E+03   0.132E+01
DrugSolver: IC,EC:    0.317E+03   0.132E+01
DrugSolver: IC,EC:    0.317E+03   0.132E+01
DrugSolver: IC,EC:    0.316E+03   0.131E+01
DrugSolver: IC,EC:    0.316E+03   0.131E+01
DrugSolver: IC,EC:    0.316E+03   0.131E+01
DrugSolver: IC,EC:    0.316E+03   0.131E+01
DrugSolver: IC,EC:    0.316E+03   0.131E+01
DrugSolver: IC,EC:    0.316E+03   0.131E+01
msg: istep,hour,IC,EC:     72   2.0   0.316E+03   0.131E+01   0.161E+01   0.200E-01
DrugSolver: IC,EC:    0.316E+03   0.131E+01
DrugSolver: IC,EC:    0.316E+03   0.131E+01
DrugSolver: IC,EC:    0.315E+03   0.131E+01
DrugSolver: IC,EC:    0.315E+03   0.131E+01
DrugSolver: IC,EC:    0.315E+03   0.130E+01
DrugSolver: IC,EC:    0.315E+03   0.130E+01
DrugSolver: IC,EC:    0.315E+03   0.130E+01
DrugSolver: IC,EC:    0.315E+03   0.130E+01
DrugSolver: IC,EC:    0.315E+03   0.130E+01
DrugSolver: IC,EC:    0.315E+03   0.130E+01
DrugSolver: IC,EC:    0.314E+03   0.130E+01
DrugSolver: IC,EC:    0.314E+03   0.130E+01
DrugSolver: IC,EC:    0.314E+03   0.130E+01
DrugSolver: IC,EC:    0.314E+03   0.130E+01
DrugSolver: IC,EC:    0.314E+03   0.130E+01
DrugSolver: IC,EC:    0.314E+03   0.130E+01
DrugSolver: IC,EC:    0.313E+03   0.129E+01
DrugSolver: IC,EC:    0.313E+03   0.129E+01
DrugSolver: IC,EC:    0.313E+03   0.129E+01
DrugSolver: IC,EC:    0.313E+03   0.129E+01
DrugSolver: IC,EC:    0.313E+03   0.129E+01
DrugSolver: IC,EC:    0.313E+03   0.129E+01
DrugSolver: IC,EC:    0.313E+03   0.129E+01
DrugSolver: IC,EC:    0.313E+03   0.129E+01
DrugSolver: IC,EC:    0.312E+03   0.129E+01
DrugSolver: IC,EC:    0.312E+03   0.129E+01
DrugSolver: IC,EC:    0.312E+03   0.129E+01
DrugSolver: IC,EC:    0.312E+03   0.129E+01
DrugSolver: IC,EC:    0.312E+03   0.129E+01
DrugSolver: IC,EC:    0.312E+03   0.129E+01
DrugSolver: IC,EC:    0.311E+03   0.128E+01
DrugSolver: IC,EC:    0.311E+03   0.128E+01
DrugSolver: IC,EC:    0.311E+03   0.128E+01
DrugSolver: IC,EC:    0.311E+03   0.128E+01
DrugSolver: IC,EC:    0.311E+03   0.128E+01
DrugSolver: IC,EC:    0.311E+03   0.128E+01
msg: istep,hour,IC,EC:     90   2.5   0.310E+03   0.128E+01   0.161E+01   0.200E-01
DrugSolver: IC,EC:    0.310E+03   0.128E+01
DrugSolver: IC,EC:    0.310E+03   0.128E+01
DrugSolver: IC,EC:    0.310E+03   0.128E+01
DrugSolver: IC,EC:    0.310E+03   0.128E+01
DrugSolver: IC,EC:    0.310E+03   0.128E+01
DrugSolver: IC,EC:    0.310E+03   0.128E+01
DrugSolver: IC,EC:    0.310E+03   0.128E+01
DrugSolver: IC,EC:    0.310E+03   0.128E+01
DrugSolver: IC,EC:    0.309E+03   0.127E+01
DrugSolver: IC,EC:    0.309E+03   0.127E+01
DrugSolver: IC,EC:    0.309E+03   0.127E+01
DrugSolver: IC,EC:    0.309E+03   0.127E+01
DrugSolver: IC,EC:    0.309E+03   0.127E+01
DrugSolver: IC,EC:    0.309E+03   0.127E+01
DrugSolver: IC,EC:    0.308E+03   0.127E+01
DrugSolver: IC,EC:    0.308E+03   0.127E+01
DrugSolver: IC,EC:    0.308E+03   0.127E+01
DrugSolver: IC,EC:    0.308E+03   0.127E+01
DrugSolver: IC,EC:    0.308E+03   0.127E+01
DrugSolver: IC,EC:    0.308E+03   0.127E+01
DrugSolver: IC,EC:    0.307E+03   0.127E+01
DrugSolver: IC,EC:    0.307E+03   0.127E+01
DrugSolver: IC,EC:    0.307E+03   0.126E+01
DrugSolver: IC,EC:    0.307E+03   0.126E+01
DrugSolver: IC,EC:    0.307E+03   0.126E+01
DrugSolver: IC,EC:    0.307E+03   0.126E+01
DrugSolver: IC,EC:    0.306E+03   0.126E+01
DrugSolver: IC,EC:    0.306E+03   0.126E+01
DrugSolver: IC,EC:    0.306E+03   0.126E+01
DrugSolver: IC,EC:    0.306E+03   0.126E+01
DrugSolver: IC,EC:    0.306E+03   0.126E+01
DrugSolver: IC,EC:    0.306E+03   0.126E+01
DrugSolver: IC,EC:    0.306E+03   0.126E+01
DrugSolver: IC,EC:    0.306E+03   0.126E+01
DrugSolver: IC,EC:    0.305E+03   0.126E+01
DrugSolver: IC,EC:    0.305E+03   0.126E+01
msg: istep,hour,IC,EC:    108   3.0   0.305E+03   0.126E+01   0.161E+01   0.200E-01
DrugSolver: IC,EC:    0.305E+03   0.126E+01
DrugSolver: IC,EC:    0.305E+03   0.126E+01
