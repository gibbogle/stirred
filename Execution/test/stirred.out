 istep  hour   IC-parent   EC-parent   IC-metab    EC-metab
     0   0.0  0.2832E+02  0.5189E+01  0.3668E+00  0.2951E-03
    18   0.5  0.2697E+03  0.2145E+01  0.1597E+01  0.1670E-01
    36   1.0  0.3145E+03  0.1505E+01  0.1611E+01  0.1965E-01
    54   1.5  0.3192E+03  0.1356E+01  0.1612E+01  0.1995E-01
    72   2.0  0.3157E+03  0.1308E+01  0.1612E+01  0.1998E-01
    90   2.5  0.3105E+03  0.1280E+01  0.1612E+01  0.1999E-01
   108   3.0  0.3049E+03  0.1256E+01  0.1612E+01  0.1999E-01
